boards/arm/faze/doc/index.rst

```diff
@@ -0,0 +1,133 @@
+
++-----------+------------+-------------------------------------+
+| Interface | Controller | Driver/Component                    |
++===========+============+=====================================+
+| NVIC      | on-chip    | nested vector interrupt controller  |
++-----------+------------+-------------------------------------+
+| SYSTICK   | on-chip    | systick                             |
++-----------+------------+-------------------------------------+
+| IOCON     | on-chip    | pinmux                              |
++-----------+------------+-------------------------------------+
+| CLOCK     | on-chip    | clock and reset control             |
++-----------+------------+-------------------------------------+
+| GPIO      | on-chip    | gpio                                |
++-----------+------------+-------------------------------------+
+| I2C       | on-chip    | i2c master/slave controller         |
++-----------+------------+-------------------------------------+
+| UART      | on-chip    | serial port-polling;                |
+|           |            | serial port interrupt               |
++-----------+------------+-------------------------------------+
+| EEPROM    | on-chip    | eeprom                              |
++-----------+------------+-------------------------------------+
+
+Connections and IOs
+===================
+
+The IOCON controller can be used to configure the LPC11U67 pins.
+
++---------+-----------------+----------------------------+
+| Name    | Function        | Usage                      |
++=========+=================+============================+
+| PIO0_2  | GPIO            | ASM2364 interrupt          |
++---------+-----------------+----------------------------+
+| PIO0_4  | I2C0            | I2C0 SCL                   |
++---------+-----------------+----------------------------+
+| PIO0_5  | I2C0            | I2C0 SDA                   |
++---------+-----------------+----------------------------+
+| PIO0_7  | I2C1            | I2C1 SCL                   |
++---------+-----------------+----------------------------+
+| PIO0_18 | UART            | USART0 RX                  |
++---------+-----------------+----------------------------+
+| PIO0_19 | UART            | USART0 TX                  |
++---------+-----------------+----------------------------+
+| PIO0_20 | GPIO            | USB sleep                  |
++---------+-----------------+----------------------------+
+| PIO1_23 | GPIO            | SSD activity white LED     |
++---------+-----------------+----------------------------+
+| PIO1_24 | I2C1            | I2C1 SDA                   |
++---------+-----------------+----------------------------+
+
+Programming and Debugging
+*************************
+
+Flashing
+========
+
+The NXP LPC11U67 MCU can be flashed by connecting an external debug probe to
+the SWD port (on-board 4-pins J2 header). In the default OpenOCD configuration
```

Comment:
Just curious but is this header actually exposed?

Reply:
Hi David. No it is not, the casing must be open and the header must be soldered. But it is actually quite easy to do.